introduction.md

```diff
@@ -1 +1,6 @@
 Myself Santa Masterting GIT and my favorite code school path is Javascript.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?